crop.go

```diff
@@ -25,31 +25,30 @@
  */
 
 /*
-package smartcrop implements a content aware image cropping library based on
+Package smartcrop implements a content aware image cropping library based on
 Jonas Wagner's smartcrop.js https://github.com/jwagner/smartcrop.js
 */
 package smartcrop
 
 import (
 	"errors"
-	"fmt"
 	"image"
 	"image/color"
-	"image/jpeg"
-	"image/png"
+	"log"
 	"math"
 	"os"
 	"time"
 
-	"code.google.com/p/draw2d/draw2d"
 	"github.com/lazywei/go-opencv/opencv"
+	"github.com/llgcode/draw2d/draw2dimg"
+	"github.com/llgcode/draw2d/draw2dkit"
 	"github.com/nfnt/resize"
 )
 
 var skinColor = [3]float64{0.78, 0.57, 0.44}
 
 const (
-	detailWeight = 0.2
+	detailWeight             = 0.2
 	faceDetectionHaarCascade = "/usr/share/opencv/haarcascades/haarcascade_frontalface_alt.xml"
 	//skinBias          = 0.01
 	useFaceDetection  = true // if true, opencv face detection is used instead of skin detection.
```

Comment:
I kinda liked it that you could see all defaults at one place, but of course I can simply move those to the default initializer.